modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`ami-0686a827d05ff9d10`
+|`rhcos-46.82.202008181646-0`
+|`ami-00b56d4237359f57e`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0ac625dd75f4875b8`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0a668d27ee6f2d05b`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0708cfc5575d806d7`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0e477c70da6751d1e`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0fedf5b459c5eee36`
+|`rhcos-45.81.202005200134-0`
+|`ami-0480eaa4de0b83565`
+|`rhcos-49.84.202107191624-0`
+|`ami-03ecefb6ad113a678`
+|`rhcos-48.84.202107202156-0`
+|`ami-0e8480cffce65c74d`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-049d5b469f54f8fb8`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-042379067546d8531`
+|`rhcos-412.86.202207142104-0-x86_64`
+
+.19+|*eu-central-1*
+|`ami-07bd2390ee0c2b2eb`
+|`rhcos-47.83.202102090044-0`
+|`ami-043bb25ed4a6b3871`
+|`rhcos-45.81.202005200134-0`
+|`ami-0baa8e69798198b4f`
+|`rhcos-48.84.202107202156-0`
+|`ami-07dc1cd741b71b502`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-072a4b61d61802fc5`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0da607360203ff018`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-05714bcec6aa9a156`
+|`rhcos-49.84.202107191624-0`
+|`ami-06b9c321d58aac43e`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0472294cbab427817`
+|`rhcos-46.82.202008071940-0`
+|`ami-06ee906c34a6bc206`
+|`rhcos-48.84.202105062123-0`
+|`ami-0459011ee17af501c`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-04c056b5a7e86668b`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-085d17a08f7991e15`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-012506a67d7285177`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0b836958d9c78e46d`
+|`rhcos-46.82.202008181646-0`
+|`ami-0dcf62208b29f32c7`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-06e39a701e2b23dd5`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0f322469661e2b01b`
+|`rhcos-45.82.202008010929-0`
+|`ami-06ea0ad4de3abe4b8`
+|`rhcos-416.94.202403071059-0-x86_64`
+
+.19+|*eu-west-1*
+|`ami-02d03a3b401bd05c6`
+|`rhcos-46.82.202008071940-0`
+|`ami-04f6587e3cce5a66f`
+|`rhcos-45.82.202008010929-0`
+|`ami-062235cc7876c4ecb`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0a574de130fd8efc3`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f175eaa2161dc2ed`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0ea57d1b0b4a20f12`
+|`rhcos-48.84.202105062123-0`
+|`ami-0e2ccf6b05ed3a339`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-08bce188c387a560e`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0015282afaac5c4b6`
+|`rhcos-45.81.202005200134-0`
+|`ami-020e22cc494ab6f27`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0d08c5f66ac8265e0`
+|`rhcos-48.84.202107202156-0`
+|`ami-0f7d388ea57cf6520`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0792d53f3a6136a14`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-00ce33bc6c6aeebf4`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-008e70e09e4a791d3`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?